collector/go.mod

```
@@ -0,0 +1,14 @@
+module github.com/open-telemetry/opentelemetry-lambda/collector
+
+go 1.14
```

```suggestion
go 1.16
```

New project so may as well